### DISCUSSION TITLE
Size limit for the custom_data inside ui_attributes file

### DISCUSSION THREAD

**Discussion by aswinc20y** [twilio-professional-services/flex-project-template#122] [Q&A] [answered]
Is there any size limit for custom_data attributes we use inside flex config ui_attributes file ?
I would like to have every department names listed inside department_selector plugins as department_options, like there would be hundreds of department_options

**Comment by dremin** [comment, marked as answer]
The ui_attributes are uploaded using the Flex Configuration API for consumption by hosted Flex, which has a limit of 50kb total config size (including the properties outside ui_attributes). If your configuration will reach near this value after adding the department data, I'd suggest looking into alternative storage options for this data, such as storing in a Twilio Runtime asset or an S3 bucket, for example.
